src/components/ArticleFooter.js

```diff
@@ -4,12 +4,17 @@ import { buildWpMobileWebUrl, canonicalizeTitle } from 'utils'
 
 export const ArticleFooter = ({ lang, title, items = [] }) => {
   const i18n = useI18n()
-  const header = i18n.i18n('suggested-articles')
+  const i18nLocale = i18n.locale
+
+  i18n.setLocale(lang)
```

Comment:
Maybe put those 2 lines in a try block and the restoring of current language in finally?